### PR TITLE
docs: Translate README to Simplified Chinese

### PR DESCRIPTION
**Pull request by Octopus058**

### CODE REVIEW

**Comment by Octopus058** [comment]
@sz3

File: README.zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+虽然解码器是一个使用手机摄像头读取动画条形码的手机应用程序：
+* 解码器安卓应用程序：https://github.com/sz3/cfc
+
+未使用互联网/蓝牙/NFC/等，所有数据都通过相机镜头传输。你可以自己试试，或者相信我的话，它真的有效。:)
+
+## 工作原理？
+
+`cimbar`是一种高密度的二维条形码格式。数据存储在彩色图块网格中——根据选择哪个图块和选择哪种颜色来绘制图块，对比特进行编码。对数据应用Reed-Solomon纠错，以解释视频->数字解码的有损性。预计错误率低于1%，并已纠正。
+
+`libcimbar`，这个优化版本，包括一个基于喷泉代码（wirheir）和zstd压缩的简单文件编码协议。高达33MB（压缩后）的文件以一系列cimbar代码编码，可以输出为图像或实时视频馈送。一旦成功解码了足够多的不同图像帧，文件将被成功重建和解压缩。即使图像接收顺序错误，或者有些图像已损坏或丢失，这也是正确的。
```
Comment (sz3) [inline README.zh-cn.md:28]: Is "wirheir" a typo?

**Comment by sz3** [comment]
Thanks for the reminder. I did have a question, if you don't mind: was this machine translated, or did you do it by hand, or machine translated + some manual edits?

I don't have a strong opinion on what the "right" way would be (there are pros/cons), just interested in how you did it.

File: README.zh-cn.md (hunk 1 above)
Comment (Octopus058) [reply README.zh-cn.md:28]: Yes, my carelessness. I'll check the translation again.

**Comment by Octopus058** [comment]
> Thanks for the reminder. I did have a question, if you don't mind: was this machine translated, or did you do it by hand, or machine translated + some manual edits?
> 
> I don't have a strong opinion on what the "right" way would be (there are pros/cons), just interested in how you did it.

Actually machine translated + manual edits. However, I may made some mistakes carelessly, I will fix them soon.

**Comment by Octopus058** [comment]
What about now?

**Comment by sz3** [comment]
Looks good, thanks for doing this!

**Review by sz3 — APPROVED** [review]

**Pull request merged**